test_scripts/MobileProjection/Phase2/009_hmiLevel_001-032.lua

```diff
@@ -0,0 +1,221 @@
+    [1] = { e = activateApp,   l = "FULL",       a = "NOT_AUDIBLE", v = "STREAMABLE" },
+    [2] = { e = deactivateApp, l = "LIMITED",    a = "NOT_AUDIBLE", v = "STREAMABLE" },
+    [3] = { e = activateApp,   l = "FULL",       a = "NOT_AUDIBLE", v = "STREAMABLE" }
+  }},
+  [28] = { t = "NAVIGATION", m = true,  s = {
+    [1] = { e = activateApp,   l = "FULL",       a = "AUDIBLE",     v = "STREAMABLE" },
+    [2] = { e = deactivateApp, l = "LIMITED",    a = "AUDIBLE",     v = "STREAMABLE" },
+    [3] = { e = activateApp,   l = "FULL",       a = "AUDIBLE",     v = "STREAMABLE" }
+  }},
+  [29] = { t = "MEDIA",      m = true,  s = {
+    [1] = { e = activateApp,   l = "FULL",       a = "AUDIBLE",     v = "NOT_STREAMABLE" },
+    [2] = { e = deactivateHMI, l = "BACKGROUND", a = "NOT_AUDIBLE", v = "NOT_STREAMABLE" },
+    [3] = { e = activateHMI,   l = "FULL",       a = "AUDIBLE",     v = "NOT_STREAMABLE" }
+  }},
+  [30] = { t = "PROJECTION", m = false, s = {
+    [1] = { e = activateApp,   l = "FULL",       a = "NOT_AUDIBLE", v = "STREAMABLE" },
+    [2] = { e = deactivateHMI, l = "BACKGROUND", a = "NOT_AUDIBLE", v = "NOT_STREAMABLE" },
+    [3] = { e = activateHMI,   l = "FULL",       a = "NOT_AUDIBLE", v = "STREAMABLE" }
+  }},
+  [31] = { t = "NAVIGATION", m = true,  s = {
+    [1] = { e = activateApp,   l = "FULL",       a = "AUDIBLE",     v = "STREAMABLE" },
+    [2] = { e = deactivateHMI, l = "BACKGROUND", a = "NOT_AUDIBLE", v = "NOT_STREAMABLE" },
+    [3] = { e = activateHMI,   l = "FULL",       a = "AUDIBLE",     v = "STREAMABLE" }
+  }},
+  [32] = { t = "DEFAULT",    m = false, s = {
+    [1] = { e = activateApp,   l = "FULL",       a = "NOT_AUDIBLE", v = "NOT_STREAMABLE" },
+    [2] = { e = deactivateHMI, l = "BACKGROUND", a = "NOT_AUDIBLE", v = "NOT_STREAMABLE" },
+    [3] = { e = activateHMI,   l = "FULL",       a = "NOT_AUDIBLE", v = "NOT_STREAMABLE" }
+  }}
+}
+
+--[[ Local Functions ]]
+local function doAction(pTC, pSS)
+  local event = pSS.e()
+  common.getMobileSession():ExpectNotification("OnHMIStatus")
+  :ValidIf(function(_, data)
+      return common.checkAudioSS(pTC, event, pSS.a, data.payload.audioStreamingState)
+    end)
+  :ValidIf(function(_, data)
+      return common.checkVideoSS(pTC, event, pSS.v, data.payload.videoStreamingState)
+    end)
+  :ValidIf(function(_, data)
+      return common.checkHMILevel(pTC, event, pSS.l, data.payload.hmiLevel)
+    end)
+end
+
+--[[ Scenario ]]
+for n, tc in common.spairs(testCases) do
+  runner.Title("TC[" .. common.getTCNum(testCases, n) .. "]: "
+    .. "[hmiType:" .. tc.t .. ", isMedia:" .. tostring(tc.m) .. "]")
+  runner.Step("Clean environment", common.preconditions)
+  runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+  runner.Step("Set App Config", common.setAppConfig, { 1, tc.t, tc.m })
+  runner.Step("Register App", common.registerApp)
+  for i = 1, #tc.s do
+    runner.Step("Action " .. i .. ",hmiLevel:" .. tc.s[i].l, doAction, { n, tc.s[i] })
```

Comment:
Propose print instead of Action .. n the name of action - activation, de-activation

Reply:
@GetmanetsIrina Done in b0c29ef